.buildkite/0_webui.yml

```
@@ -0,0 +1,8 @@
+# This file represents what is put into the webUI.
+# It is purely for keeping track of the changes we make to the webUI configuration; modifying this file has no effect.
+steps:
+  - label: ":buildkite: Pipeline upload"
```

Note that you also need to copy these changes and paste them into the "Edit Steps" section of the Buildkite web UI for them to take effect.

```suggestion
  - label: ":buildkite: Pipeline upload"
    plugins:
      - JuliaCI/merge-commit#v1: ~
```